blueprint/api.apib

```diff
@@ -2534,10 +2594,34 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(array[Stripe Card Resource])
 + include JSON API Version
 
+## Stripe Customer Resource (object)
++ include Stripe Customer Resource Identifier
++ attributes(Stripe Customer Attributes)
++ relationships
+    + subscriptions
+      + data(Stripe Subscription Resource Identifier)
+    + user
+      + data(User Resource Identifier)
+
 ## Stripe Customer Resource Identifier (object)
 + id: `1` (string, required)
 + type: `stripe-customer`
 
+## Stripe Customer Attributes (object)
++ created: `016-07-08T03:03:51.967Z` (string) - A timestamp, indicating when the customer was created by Stripe
++ currency: `usd` (string)
++ delinquent: `true` (boolean) - Whether or not the latest charge for the customer’s latest invoice has failed.
++ email: `johndoe@email.com` (string)
++ `id-from-stripe`: `cus_9SBK9vXBkZEdfe` (string) - The ID that Stripe assigned the customer.
```

Comment:
It already has `created`, should it be `created-at` instead or should it have both `inserted-at` and `created-at`? Created at being when it was created by stripe and inserted being when it was received by the code corps database?